src/components/map/MapService.js

```diff
@@ -752,7 +752,8 @@ goog.require('ga_urlutils_service');
               }
 
               if (options.zoomToExtent) {
-                olMap.getView().fit(olLayer.getExtent(), olMap.getSize());
+                var extent = olLayer.getSource().getExtent();
```

Comment:
The result of `layer.getExtent()` is `undefined` while the result of `source.getExtent()` is a correct array.